packages/fluentui/react-telemetry/src/useTelemetryData.ts

```diff
@@ -0,0 +1,65 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+import { ComponentPerfStats, Telemetry } from '@fluentui/react-bindings';
+
+export type TelemetryDataItem = ComponentPerfStats & {
+  componentName: string;
+  msStylesTotal: number;
+};
+
+export type TelemetryDataTotals = Omit<TelemetryDataItem, 'componentName'>;
+
+export function useTelemetryData(
+  telemetry: Telemetry,
+  id: number,
```

Comment:
`id` is not a good name, I had to go to usages to understand its meaning.
`key` or `tick` or ?